packages/style-engine/phpunit/class-wp-style-engine-css-declarations-test.php

```diff
@@ -149,14 +149,37 @@ public function test_generate_prettified_with_more_indents_css_declarations_stri
 	 */
 	public function test_remove_unsafe_properties_and_values() {
 		$input_declarations = array(
-			'color'        => '<red/>',
+			'color'        => 'url("https://wordpress.org")',
```

Comment:
I've changed this one to be a value that `safecss_filter_attr` strips, instead, since this PR now removes the call to `esc_html` so we no longer need to test whether or not HTML characters will be escaped.